Advanced-Recruit-Training/src/Recruit.pro

```diff
@@ -0,0 +1,25 @@
+# Created by and for Qt Creator. This file was created for editing the project sources only.
```

Comment:
Oooh! This is why the travis build was failing! This `.pro` file was generated somehow, so now when Travis tried to do qmake, he didn't know whether to run on this file, or the `BatteryLife.pro`! 
Remove this guy, and we should be good to go.